web/themes/custom/accessibility_theme/src/patterns/components/hero/hero.twig

```diff
@@ -0,0 +1,53 @@
+{{ attach_library('accessibility_theme/hero') }}
+
+<div class="hero {{ modifier }}" {{ extra_attributes }}>
```

Comment:
Let's remove the extra_attributes variable, please.  Let's also remove the commented block below and use the method used in our training: https://mariohernandez.gitbook.io/components/chapter-3/drupal-attributes